src/AiClient.php

```diff
@@ -114,14 +115,44 @@ public static function defaultRegistry(): ProviderRegistry
     /**
      * Checks if a provider is configured and available for use.
      *
+     * Supports multiple input formats for developer convenience:
+     * - ProviderAvailabilityInterface: Direct availability check (backward compatible)
```

Comment:
This isn't purely for backward compatibility, we just want to support all 3 ways.
```suggestion
     * - ProviderAvailabilityInterface: Direct availability check
```

Reply:
resolved https://github.com/WordPress/php-ai-client/pull/88/commits/e14a7567b10cb655dd4a90108b465d14567541e5